core/src/main/scala/com/rackspace/com/papi/components/checker/util/IdentityTransformPool.scala

```diff
@@ -29,8 +32,16 @@ object IdentityTransformPool extends Instrumented  {
   //
   private val tf = TransformerFactory.newInstance("org.apache.xalan.xsltc.trax.TransformerFactoryImpl", this.getClass.getClassLoader)
   private val pool = new SoftReferenceObjectPool[Transformer](new IdentityTransformerFactory(tf))
-  private val activeGauge = metrics.gauge("Active")(numActive)
-  private val idleGauge = metrics.gauge("Idle")(numIdle)
+  Try {
+    metrics.gauge("Active")(numActive)
+  } recover {
+    case e: RuntimeException => logger.info("Problem adding new Active gauge metric.", e)
```

Comment:
Same here WARN...and confirm that the gauge is still functional afterwards.  Because I'm afraid this will happen not just during testing.  Is it possible to happen with multiple validators??